apps/archive/common.py

```diff
@@ -428,6 +428,10 @@ def item_schema(extra=None):
                 'marked_for_legal': {
                     'type': 'boolean',
                     'default': False
+                },
+                'marked_archived_only': {
```

Comment:
Since this attribute exists only in `archived` repo, it should be moved to `ArchivedResource` instead of `ArchiveResource`